README.md

```diff
@@ -55,6 +63,9 @@ Write your code here...
 
 ```ruby
 # code here
+own.delete("Belle")
+
+guests.push("Belle")
 ```
```

Comment:
You were close. You select the town, but, need to go a level deeper and also select residents to remove Belle. To add Belle you have to remember to select the castle as well as the town.
Answer:
```
town[:residents].delete("Belle")
town[:castle][:guests] << "Belle"
```